engine_wrapper.py

```diff
@@ -1,7 +1,35 @@
+import os
 import chess
 import chess.xboard
 import chess.uci
 
+class EngineFactory:
+    def __init__(self, config):
+        self.config = config
+    def __call__(self, board):
```

Comment:
Great demonstration that "factory" objects are in fact just partially applied functions #haskell

Reply:
Yes, and I could have written it that way in python even.

Reply:
https://github.com/lakinwecker/lichess-uci-bot/commit/b91b7a19b998ca19e60fc530ca27eb3500e5ca16